### ISSUE TITLE
ALL ENTRIES LOST

### ISSUE THREAD

**Issue by juanmduran** [JabRef/jabref#8746]
### JabRef version

5.6 (latest release)

### Operating system

macOS

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Change an entry [Title] from " to `
2. Unable to save
3. Quit and open again. 
4. All entries after that change are lost. The modified entry is half there. Anything after "title =" is gone
5. Very dissapointed6. 
6. Deleting JabRef and finding a new bib manager.

### Appendix

...
<details>

<summary>Log File</summary>

```
Paste an excerpt of your log file here
```

</details>

**Comment by koppor** [comment]
Thank you for the report! 

Internal check: There should have been a .sav file.

**Comment by Siedlerchr** [comment]
@juanmduran Thanks for the report. That doesn't sound good.  I tried to reproduce this, can you please tell us where exactly you modified the title? In the title field or in the  BibTeX source?

**Comment by juanmduran** [comment]
Hi, 

It isn't good. I lost entries, keywords, and groups. I updated to 5.6, and changed the title for an entry (again, from "XXXX" to `XXX'). Since JabRef wouldn't save the change (I don't remember the reported error), I restarted it. Everything up to that entry is gone.

The .sav file does not have all the entries (although it has more than the current .bib) but no keywords no groups nothing. I've lost years of work. This is literal since I didn't have a copy and the .bak was also truncated together with the .bib.
